### PR TITLE
Adds delete functions to setup_perf_cluster.sh script

### PR DESCRIPTION
**Pull request by danehans**
This PR adds support for uninstalling what was done by `setup_all` in `setup_perf_cluster.sh` script.

Fixes https://github.com/istio/istio/issues/3023

### CODE REVIEW

**Comment by danehans** [comment]
Supersedes https://github.com/istio/istio/pull/3024

//cc @ldemailly

File: tools/setup_perf_cluster.sh [hunk 1]
```diff
@@ -113,6 +128,12 @@ function install_istio() {
   Execute sh -c 'sed -e "s/_debug//g" install/kubernetes/istio-auth.yaml | egrep -v -e "- (-v|\"2\")" | kubectl apply -f -'
 }
 
+# assumes run from istio/ (or release) directory
+function delete_istio() {
+  # Use the non debug ingress and remove the -v "2"
+  Execute sh -c 'sed -e "s/_debug//g" install/kubernetes/istio-auth.yaml | egrep -v -e "- (-v|\"2\")" | kubectl delete -f -'
```
Comment (ldemailly) [inline tools/setup_perf_cluster.sh]: I think 
```
kubectl delete -f install/kubernetes/istio-auth.yaml
```
would work the same as the image and flags don't matter during delete

File: tools/setup_perf_cluster.sh [hunk 2]
```diff
@@ -238,6 +259,15 @@ function setup_all() {
   setup_cluster_all
 }
 
+function delete_all() {
+  echo "Deleting Istio mesh, cluster $CLUSTER_NAME, Instance $VM_NAME and firewall rules for project $PROJECT in zone $ZONE"
+  sleep 3
```
Comment (ldemailly) [inline tools/setup_perf_cluster.sh]: add
echo "Interrupt now if you don't want to delete..."
and sleep 5s instead of 3

**Comment by ldemailly** [comment]
thank you !

**Pull request merged**